tests/integration/hive_metastore/test_pipeline_migrate.py

```diff
@@ -0,0 +1,156 @@
+from datetime import timedelta
+
+from databricks.sdk.service.jobs import Task, PipelineTask
+from databricks.sdk.service.pipelines import NotebookLibrary, PipelineLibrary, PipelineCluster, PipelineState
+from databricks.sdk.service.workspace import Language
+
+from databricks.labs.ucx.hive_metastore.pipelines_migrate import PipelinesMigrator
+
+from ..assessment.test_assessment import _PIPELINE_CONF
+
+_TEST_STORAGE_ACCOUNT = "storage_acct_1"
+_TEST_TENANT_ID = "directory_12345"
+
+
+def test_pipeline_migrate(
+    ws, make_pipeline, make_random, watchdog_purge_suffix, make_directory, runtime_ctx, make_mounted_location
+):
+    src_schema = runtime_ctx.make_schema(catalog_name="hive_metastore")
+    dlt_notebook_paths = [f"{make_directory()}/dlt_notebook_1", f"{make_directory()}/dlt_notebook_2"]
+
+    src_tables = [
+        runtime_ctx.make_table(catalog_name="hive_metastore", schema_name=src_schema.name, non_delta=True),
+        runtime_ctx.make_table(
+            schema_name=src_schema.name,
+            external_csv=make_mounted_location,
+            columns=[("`foobar`", "STRING")],
+        ),
+    ]
+    runtime_ctx.make_notebook(
+        content=f"""create or refresh streaming table st1\nas select * from stream(hive_metastore.{src_schema.name}.{src_tables[0].name})""".encode(
+            "ASCII"
+        ),
+        path=dlt_notebook_paths[0],
+        language=Language.SQL,
+    )
+    runtime_ctx.make_notebook(
+        content=f"""create or refresh streaming table st2\nas select * from stream(hive_metastore.{src_schema.name}.{src_tables[1].name})""".encode(
+            "ASCII"
+        ),
+        path=dlt_notebook_paths[1],
+        language=Language.SQL,
+    )
+
+    pipeline_names = [
+        f"pipeline-{make_random(4).lower()}-{watchdog_purge_suffix}",
```

Comment:
Shouldn't this be part of the fixture?

Reply:
I needed the list of names to later assert via name (as we rename the HMS pipelines). Also the pipelines creation does not return a name in CreatePipelineResponse just the id.